VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs

```diff
@@ -348,13 +349,30 @@ private void CheckParticipantsAreStillConnected()
         {
             var loggedInParticipants = LoggedInParticipants(_browsers.Keys, _c.Test.ConferenceParticipants);
             var participantDetailsResponses = loggedInParticipants as ParticipantDetailsResponse[] ?? loggedInParticipants.ToArray();
+
+            By panelElement = null;
+
             foreach (var user in participantDetailsResponses)
             {
                 if ((user.UserRole == UserRole.Judge)) continue;
-                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeParticipantPanel.ParticipantStatus(user.Id));
-                _browsers[_c.CurrentUser].ScrollTo(JudgeParticipantPanel.ParticipantStatus(user.Id));
-                _browsers[_c.CurrentUser].TextOf(JudgeParticipantPanel.ParticipantStatus(user.Id)).ToUpper()
+
+                if (user.HearingRole.Equals("Observer", StringComparison.OrdinalIgnoreCase))
+                {
+                    panelElement = JudgeParticipantPanel.ObserverStatus(user.Id);
+                }
+                else if (user.HearingRole.Equals("Panel Member", StringComparison.OrdinalIgnoreCase))
+                {
+                    panelElement = JudgeParticipantPanel.PanelMemberStatus(user.Id);
+                }
+                else
+                {
+                    panelElement = JudgeParticipantPanel.ParticipantStatus(user.Id);
+                }
+
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(panelElement, _maxTimeout);
+                _browsers[_c.CurrentUser].TextOf(panelElement).ToUpper()
                     .Should().BeOneOf("CONNECTED", "IN CONSULTATION");
```

Comment:
Can you make a practice of adding exception message to this?  "Expected waiting room status for {currentUser} to be either Connected or in cunsultation"